src/main/java/com/muzika/homeworksb/service/impl/TodoServiceImpl.java

```diff
@@ -0,0 +1,132 @@
+package com.muzika.homeworksb.service.impl;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.muzika.homeworksb.dto.TaskHistoryResponseDto;
+import com.muzika.homeworksb.dto.TodoCreateDto;
+import com.muzika.homeworksb.dto.TodoResponseDto;
+import com.muzika.homeworksb.dto.TodoUpdateDto;
+import com.muzika.homeworksb.exception.EntityNotFoundException;
+import com.muzika.homeworksb.mapper.TaskHistoryMapper;
+import com.muzika.homeworksb.mapper.TodoMapper;
+import com.muzika.homeworksb.model.TaskHistory;
+import com.muzika.homeworksb.model.Todo;
+import com.muzika.homeworksb.repository.TaskHistoryRepository;
+import com.muzika.homeworksb.repository.TodoRepository;
+import com.muzika.homeworksb.service.TodoService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@RequiredArgsConstructor
+@Service
+public class TodoServiceImpl implements TodoService {
+
+    private final TodoRepository todoRepository;
+    private final TaskHistoryRepository taskHistoryRepository;
+    private final TodoMapper todoMapper;
+    private final TaskHistoryMapper historyMapper;
+
+    @Override
+    public TodoResponseDto save(TodoCreateDto createDto) {
+
+        Todo todo = todoMapper.toModel(createDto);
+        todo.setCreatedDate(LocalDateTime.now());
```

Comment:
You can use annotation `@CreationTimestamp`. See more: https://www.baeldung.com/hibernate-creationtimestamp-updatetimestamp